src/lib/systemlib/system_params.c

```diff
@@ -111,8 +111,8 @@ PARAM_DEFINE_INT32(SYS_MC_EST_GROUP, 2);
 /**
  * TELEM2 as companion computer link (deprecated)
  *
- * This parameter is deprecated. Do not change it, use the more generic serial
- * configuration parameters instead.
+ * This parameter is deprecated (from PX4 v1.8.1). Use the generic serial
```

Comment:
v1.8.1 does not include the new ones yet, only the next release will (v1.9).

Reply:
OK. From a docs point of view then we could either ignore this OR modify slightly to:

```
* This parameter is deprecated and will be removed in 1.9.0. Use the generic serial
* configuration parameters instead (e.g. MAV_0_CONFIG, MAV_0_MODE, etc.).
```

Reply:
Right, we can update to `will be removed after 1.9.0` (*in* would mean to me that it's already removed with 1.9.0).

Reply:
Done (and yes, you're right about the use of "in vs. after"). Squash and merge at will!